### PR TITLE
Checkout books

### PR DESCRIPTION
**Pull request by cassiewallace**
Changes:
- Add ability to checkout books
- Add ability for admin to manage users
- Limit access to authenticated users
- Update pagination
- Re-add login to admin panel

### CODE REVIEW

File: api/urls.py [hunk 1]
```diff
@@ -9,5 +9,6 @@
 router.register(r'users', user_views.UserViewSet)
 
 urlpatterns = [
-    path('', include(router.urls)),
+    url(r'^', include(router.urls)),
```
Comment (quentinR) [inline api/urls.py]: you should use path instead of url, url is deprecated

File: books/models.py [hunk 2]
```diff
@@ -4,8 +4,10 @@
 class Book(models.Model):
     title = models.CharField(max_length=125, null=False)
     author = models.CharField(max_length=125, null=False)
-    creator = models.ForeignKey('auth.User', related_name='books', 
-        on_delete=models.CASCADE)
+    creator = models.ForeignKey('auth.User', on_delete=models.CASCADE,
```
Comment (quentinR) [inline books/models.py]: Like we discussed, i think owner is a better name

File: users/views.py [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
-class UserViewSet(viewsets.ReadOnlyModelViewSet):
+class UserViewSet(viewsets.ModelViewSet):
     """
-    Provides `list` and `detail` actions for Users
+    Provides `list`, `create`, `retrieve`, `update`, and `destroy` 
+    actions for users.
     """
     queryset = User.objects.all()
     serializer_class = UserSerializer
+    permission_classes = [permissions.IsAdminUser]
```
Comment (quentinR) [inline users/views.py]: You could use decorator instead

Comment (cassiewallace) [reply users/views.py]: will change with next PR, with all the permissions updates

**Pull request merged**